ui5conblr/index.html

```diff
@@ -106,181 +105,353 @@
+						</svg>
+					<div class="b-font size_42 weight_light b-ui margin_30">explore.</div>
+					<div class="b-font size_18">UI5con is a free of charge event for UI5 enthusiasts to meet in person and find out what’s happening in the UI5 world.</div>
+				</div>
+				<div class="b-list__item">
+					<svg class="b-list__icon" x="0px" y="0px" viewBox="0 0 80 80" style="enable-background:new 0 0 80 80;" xml:space="preserve">
+							<path d="M35.9,44.1c0.8,0.5,1.1,1.6,0.6,2.4c-0.1,0.2-0.3,0.4-0.6,0.6c-0.4,0.5-1,0.8-1.6,0.8s-1.1-0.3-1.3-0.8l-6.4-6.4L20.2,47l-7.5-7.3c-2.7-2.5-4.2-6-4.2-9.7c0-3.6,1.6-7,4.2-9.5l2.4-2.6L3.6,6.8l3.8-3.1L18.3,15l2-2.3c2.4-2.6,5.8-4.1,9.3-4c3.7,0,7.2,1.4,9.8,4.1l7.3,8L40.5,27l1.9,1.7l4.2,4.4c0.5,0.4,0.8,1,0.8,1.7c0,0.6-0.3,1.2-0.8,1.5c-0.5,0.7-1.6,0.9-2.3,0.3c-0.1-0.1-0.2-0.2-0.3-0.3l-6.7-6.4l-8,8.2l1.8,1.6L35.9,44.1z M39.5,20.4l-4-4.2c-1.6-1.6-3.8-2.5-6-2.4c-2.1,0-4,0.9-5.4,2.4l-7.7,8c-1.6,1.6-2.6,3.6-2.8,5.9c0.1,2.3,1.1,4.5,2.8,6l3.8,3.6L39.5,20.4z M71.9,68.7l4.9,4.7l-3.1,3.3l-7.7-7l-3.3,3.1C60.3,75.4,57,77,53.4,77c-3.5,0-6.8-1.6-9-4.2l-8.2-7.7l26.4-26.4l8.2,7.9c5.1,4.7,5.5,12.6,0.8,17.8c-0.3,0.3-0.5,0.6-0.8,0.8l-1.3,1.1L71.9,68.7z M67.2,61.3c1.6-1.4,2.5-3.4,2.4-5.5c0.1-2.1-0.7-4-2.3-5.4L62.8,46l-19,18.9L48,69v0.2h0.2c1.3,1.6,3.2,2.6,5.2,2.6c2.1-0.1,4.1-1,5.5-2.6L67.2,61.3z"/>
+						</svg>
+					<div class="b-font size_42 weight_light b-ui margin_30">connect.</div>
+					<div class="b-font size_18">UI5con is meant to connect all the geeks out there &mdash; have a common platform to get to know each other, discuss innovations and find solutions.</div>
+				</div>
+			</div>
+		</div>
 
-					</div>
-					<div class="b-font size_18">
-						<ul>
-							 <li><b>We start registrations at 8am, sessions from 9AM to 6PM followed with evening special to wind up by 7PM.</b> </li>
-							<li><b>Registrations start at 8AM</b> </li>
-							<li><b>Bring your own laptops for hands-on sessions</b> </li>
-						</ul>
-					</div>
 
-					<br>
 
-					<div class="b-font size_18">
-						See below in Travel information for detailed info how to get there.
+		<div class="b-ui b-font size_18 align_center ">
+			<span class="b-font weight_bold">SPONSORING:</span> For the very first time, you can contribute to this event by becoming a sponsor. To discuss opportunities, please <a href="mailto:openui5@sap.com?subject=%5BUI5con%20Sponsoring%5D">contact us</a>.
+		</div>
+	</div>
+	<div class="b-section mod_info">
+		<div id="section-featured-speakers" class="b-font size_36 b-ui align_center color_black margin_20_0_30 grid-col--12">
+			FEATURED SPEAKERS
```

Comment:
Not really. Those are the Speakers for Germany.

Ok, maybe this is all commented out - I can't easily see this in this GitHub review mode. But that's all stuff from Germany here, which I don't want to review.